resources/language/resource.language.fr_fr/strings.po

```diff
@@ -574,23 +574,23 @@ msgstr "Désactiver la notification au démarrage"
 
 msgctxt "#30142"
 msgid "Custom"
-msgstr "Personalisé"
+msgstr "Personalisée"
 
 msgctxt "#30143"
 msgid "Custom view ID"
-msgstr "ID de la vue personalisée"
+msgstr "Numéro de la vue personalisée"
```

Comment:
change "Numéro" with "ID"

Reply:
Ok, will fix it in my next commit.

Reply:
"Identifiant" seems more appropriate as "ID" isn't common in french.

Reply:
> "Identifiant" seems more appropriate as "ID" isn't common in french.

I like the idea, thank you for the suggestion :)

Is it fine for you @CastagnaIT?

Reply:
yes it is good